components/Features/FeatureForm.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styles from './FeatureForm.module.scss';
+
+export default function FeatureForm() {
+  return (
+    <div className={styles['form-layout']}>
+      <div className={styles['form-block']}>
+        <h2 className={styles['form-header']}>
+          Increase your
+          <br />
+          charity donations
+        </h2>
+        <div className={styles['form-text']}>
+          Leave your contact information below, and we&apos;ll be in touch soon
+          to help you get started
+        </div>
+        <form
+          id="contact-form"
+          name="email-form"
+          className={styles.form}
+          method="post"
+        >
+          <label htmlFor="name-2" className={styles['field-label']}>
```

Comment:
`name-2` doesn't really explain what this field does

Reply:
Also I recommend looking at these semantic components: https://react.semantic-ui.com/collections/form/

Then we won't need so much custom CSS that we manage. You could probably cut this file down significantly.